Documentation/guides/building-apps/build-properties.md

```diff
@@ -965,7 +965,8 @@ in their `.csproj`. Alternatively provide the property on the command line:
 /p:AndroidUseAapt2=True
 ```
 
-Added in Xamarin.Android 8.3.
+Added in Xamarin.Android 8.3. `AndroidUseAapt2=false` is deprecated in
```

Comment:
"Convention" usage question: should this be "`$(AndroidUseAapt2)=false` is deprecated…", adding the `$()`-fu?

*Is* there a convention here, in public docs?

Reply:
When I first looked at it, I figured it was good enough not to worry about it too much. But now that the question is raised, I think the choice that would be most consistent with other general MSBuild docs would be something like:

```
Setting `AndroidUseAapt2` to `false` is deprecated...
```

References:

- In https://docs.microsoft.com/visualstudio/msbuild/common-msbuild-project-properties and https://docs.microsoft.com/dotnet/core/project-sdk/msbuild-props, the properties are always plain and not decorated with `$()`.

  Example quotation:

  > For example, if you want access to preview code analyzers, set `AnalysisLevel` to `preview`.

- In https://docs.microsoft.com/visualstudio/msbuild/msbuild-properties and https://docs.microsoft.com/visualstudio/msbuild/comparing-properties-and-items, the usage of `$()` is called out specifically, and the `$()` syntax is only used when referring to the properties in the context of retrieving values in a project file.

  For example, the [_Store XML in properties_](https://docs.microsoft.com/visualstudio/msbuild/msbuild-properties?view=vs-2019#store-xml-in-properties) section contains both syntaxes that reflect slightly different usages:

  > Properties can contain arbitrary XML, which can help in passing values to tasks or displaying logging information. The following example shows the `ConfigTemplate` property, which has a value that contains XML and other property references. MSBuild replaces the property references by using their respective property values. Property values are assigned in the order in which they appear. Therefore, in this example, `$(MySupportedVersion)`, `$(MyRequiredVersion)`, and `$(MySafeMode)` should have already been defined.